components/radio/stories/radio.stories.js

```diff
@@ -91,31 +95,132 @@ export default {
 	},
 };
 
-export const Default = (args) => html`
+// ===== Story specific templates ======
+
+// @todo Radio needs more robust coverage of all states, including hover, focus, etc.
+const ChromaticGroupTemplate = (args) => html`
 	<div style="display: flex; flex-direction: column; align-items: flex-start;">
 		${Template({
 			...args,
-			label: "Default"
+			label: "Default, not selected",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-1",
 		})}
 		${Template({
 			...args,
-			isEmphasized: true,
 			isChecked: true,
-			label: "Emphasized radio button label that is so long it has to wrap",
+			isEmphasized: false,
+			label: "Default, selected",
 			customStyles: {
 				"max-width": "220px",
-			}
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-2",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			isEmphasized: true,
+			label: "Emphasized, selected",
+			customStyles: {
+				"max-width": "220px",
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-3",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
-			label: "Disabled"
+			label: "Disabled",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-4",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
 			isReadOnly: true,
-			label: "Read only"
+			label: "Read-only",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-5",
 		})}
 	</div>
 `;
+
+const BasicGroupTemplate = (args) => html`
+	<div style="display: flex; flex-direction: column; align-items: flex-start;">
+		${Template({
+			...args,
+			label: "Example label",
```

Comment:
I've updated the Default story that appears in Storybook to only have the single radio template, instead of the two radios template; the control to change the label now works with that. The two radio template has set labels to include testing of wrapping text.